packages/flutter/lib/src/widgets/banner.dart

```diff
@@ -236,7 +264,9 @@ class CheckedModeBanner extends StatelessWidget {
       result = new Banner(
         child: result,
         message: 'SLOW MODE',
-        location: BannerLocation.topRight);
+        textDirection: TextDirection.ltr,
+        location: BannerLocation.topRight,
```

Comment:
You're right here.  Maybe set the correct location (topLeft) in RTL?

Reply:
https://github.com/flutter/flutter/issues/11905